introduction.md

```diff
@@ -1 +1,4 @@
 Hi, everybody! Hi, Dr. Nick! Favorite Code School path is Electives.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?